leo_docking/launch/dock_rover.launch.py

```diff
@@ -24,7 +24,7 @@ def generate_launch_description():
 
     aruco_tracker = Node(
         package="aruco_opencv",
-        executable="aruco_tracker",
+        executable="aruco_tracker_autostart",
```

Comment:
Docstring above is incorrect.

Also would be nice to use `os.path.join` for the param_dir